Attributes/Troops/Logic/attribute_5.gd

```diff
@@ -0,0 +1,29 @@
+extends TroopAttribute
+
+# Logic for splash attribute
+
+func build_action() -> Action:
+    var action: Action = Action.new()
+    action.name = attribute.name
+    action.description = attribute.description
+    action.setup(parent.game, on_attack)
+    return action
+
+#loops to all tiles around defender and applies half damage
+
+func on_attack(defender: Unit):
+    var defender_pos = defender.pos
+    var splash_tiles = _get_surrounding(defender_pos, 1)
```

Comment:
`_get_surrounding` is not declared in this code, so must change to `defender._get_surrounding` because it inherits from `troop.gd`